tests/assets/suc.yaml

```diff
@@ -9,13 +9,14 @@ metadata:
 spec:
   concurrency: 1
   #version:  latest
-  version: "opensuse-v1.23.5-44"
+  version: "v2.3.1-k3sv1.25.11-k3s1"
```

Comment:
Actually this test is doing a downgrade. Because what we build here is usually newer than any released image. Without changing this, it doesn't matter much to which version we downgrade too. If this breaks we just downgrade to a newer one (if that's the problem at all).

I don't like this much but I'm only trying to make the test actually test something. It needs further improvement to make more sense.